README.md

```diff
@@ -38,13 +41,156 @@ class Translations {
 }
 ```
 
-### Flavors
+Did you notice the `@GenerateArb()` annotation? That where the magic come from.
+
+Once you run you favorite build_runner command it will generate the ARB corresponding to this class.
+
+This annotation can be customize with the following fields:
 
-This package can help you deal with flavor, list the flavors on `GenerateIntl` annotation and provide an ARB file for each locales you support
+- defaultLocale: locale your working with, default to 'en'
+- defaultFlavor: default flavor of your project, default to 'default'
+- suppressMetaData: suppress meta data when generating the ARB file, default to false
+- suppressLastModified: suppress last modified when generating the ARB file, default to false
+- includeSourceText: whether to include source_text in messages, default to false
+- dir: directory where the ARB should be generated
+
+## Flutter project
+
+If you have done Flutter project with localizations before, you'll need to declare a LocalizationDelegate in your app.
```

Comment:
If you have an existing Flutter project with localization support, then you'll need to declare a LocalizationDelegate in your app.